### PR TITLE
Improvements to remaining lessons

### PR DESCRIPTION
**Pull request by iamvery**

### CODE REVIEW

**Comment by wolfbrain** [comment]
:eyes: @killpack

File: about_enums.exs [hunk 1]
```diff
@@ -53,227 +48,220 @@ defmodule AboutEnums do
...
   end
 
-  think "find takes a default" do
+  think "Find takes a default" do
     list = [1, 2, 3]
     assert Enum.find(list, 4, fn (x) -> x > 3 end) == __?
   end
 
-  think "what index is this number at?" do
+  think "What index is this number at?" do
```
Comment (killpack) [inline about_enums.exs]: While you're in here, might as well try and avoid the dangling preposition. `think  "What is the index of this number?"`?

File: about_enums.exs [hunk 2]
```diff
@@ -53,227 +48,220 @@ defmodule AboutEnums do
...
   end
 
-  think "each element with its index" do
+  think "Get each element with its index" do
     list = [:a, :b, :c]
     assert Enum.with_index(list) == __?
   end
 
-  think "enums can be chunked" do
+  think "Chunking elements into group" do
```
Comment (killpack) [inline about_enums.exs]: group**s**?

File: about_lists.exs [hunk 3]
```diff
@@ -11,23 +11,23 @@ defmodule AboutLists do
...
     assert [1, 2, 3] -- [2] == __?
   end
 
-  think "Only first element is removed with truncate operator" do
+  think "Only the first element is removed with truncate operator" do
     assert [:foo, :bar, :foo] -- [:foo] == __?
   end
 
-  think "Truncate operator do nothing when element not in list" do
+  think "The truncate operator does nothing when element not in list" do
```
Comment (killpack) [inline about_lists.exs]: "when **an** element **is** not in **the** list"?

File: about_named_functions.exs [hunk 4]
```diff
@@ -5,23 +5,22 @@ defmodule AboutNamedFunctions do
...
+  think "Calling  a named function" do
     assert hello("world!") == __?
   end
 
   def hello(name, country) do
     "Hello #{name} from #{country}"
   end
 
-  think "Functions are identified by name and number of parameter" do
+  think "A function with the same name, but different argument is a different function" do
```
Comment (killpack) [inline about_named_functions.exs]: "different arity", perhaps? or "different number of arguments"?

File: about_named_functions.exs [hunk 5]
```diff
@@ -5,23 +5,22 @@ defmodule AboutNamedFunctions do
     "Hello #{name}"
   end
 
-  think "Declaring and using a named function" do
-
+  think "Calling  a named function" do
```
Comment (killpack) [inline about_named_functions.exs]: You've got an extra space after "Calling".

File: about_tuples.exs [hunk 6]
```diff
@@ -11,15 +11,9 @@ defmodule AboutTuples do
...
-    # Note: Elixir provides access to erlang primitives with atom :erlang
-    quote do: :erlang.element(unquote(index), unquote(tuple))
-  end
-
-  think "In erlang tuples and lists start at index 1, which is different in elixir" do
+  think "In erlang tuples and lists start at index 1, which is different in Elixir" do
     a_tuple = {:foo, :bar}
-    foo_index = 0
-    assert elem(a_tuple, foo_index) == erlang_elem(a_tuple, foo_index + __?)
+    assert elem(a_tuple, 0) == :erlang.element(__?, a_tuple)
```
Comment (killpack) [inline about_tuples.exs:16]: I like this much better than all the quote / unquote stuff.

File: about_tuples.exs [hunk 7]
```diff
@@ -54,25 +48,25 @@ defmodule AboutTuples do
...
     # Note: unquote is the reverse of quote
     # It gives a block from its representation
     unquoted_block = unquote {:"{}", [], [1, 2, 3]}
-    # Note: it's an hungarian notation
-    # the atom :"{}" represent the function
-    # the list [] contains metadatas like the line and module where code is defined
+    # Note: it's in hungarian notation
+    # the atom :"{}" represents the function
+    # the list [] contains metadata like the line and module where code is defined
     # the list [1, 2, 3] are arguments passed to the function
```
Comment (killpack) [inline about_tuples.exs]: Maybe "the list **contains** arguments"? "the list are arguments" sounds wrong.

**Comment by killpack** [comment]
👌

File: about_tuples.exs (hunk 6 above)
Comment (iamvery) [inline about_tuples.exs:16]: Ya I think all that was done to avoid the arguments being a different order :laughing:

**Comment by iamvery** [comment]
Thanks for all the grammar sanity checks. Seems the original author wasn't a native english speaker. Will fix 'em!

**Pull request merged**